metricbeat/module/windows/_meta/config.yml

```diff
@@ -0,0 +1,13 @@
+- module: windows
+  metricsets: ["perfmon"]
+  enabled: true
+  period: 1s
+  counters:
```

Comment:
In source i then have to write this `config:"perfmon.counters" validate:"required"`?